vms/avm/txs/txstest/context.go

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package txstest
+
+import (
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/snow"
+	"github.com/ava-labs/avalanchego/vms/avm/config"
+	"github.com/ava-labs/avalanchego/wallet/chain/x/builder"
+)
+
+func newContext(
```

Comment:
nit: I don't think we need this function since it's only used in one place + this is a type from a downstream package anyways, but leaving it to your preference if you want to keep this or not.

Reply:
just keeping the same structure as P-chain (we also call newContext once there)